### ISSUE TITLE
Update to `pkgdown` in the CI workflow generates new warnings and errors

### ISSUE THREAD

**Issue by Puzzled-Face** [openpharma/crmPack#852]
The version of the `pkgdown` package used in our CI workflows has been upgraded to 2.1.0.  This has caused many warnings - mainly related to accessibility - to be newly generated.  See [here](https://www.tidyverse.org/blog/2024/07/pkgdown-2-1-0/#user-experience) for details of the changes.

**_This potentially blocks every other pending PR._**

Typical warnings are 

```
── Sitrep ──────────────────────────────────────────────────────────────────────
✖ URLs not ok.
  In DESCRIPTION, URL is missing package url
  (https://openpharma.github.io/crmPack).
  See details in `vignette(pkgdown::metadata)`.
✖ Favicons not ok.
  Found package logo but not favicons.
  Do you need to run `build_favicons()`?

<output deleted>

✖ Icon "fa-github" lacks an `aria-label`.
ℹ Specify `aria-label` to make the icon accessible to screen readers.
ℹ Learn more in `vignette(pkgdown::accessibility)`.

<output deleted>

── Building articles ───────────────────────────────────────────────────────────
Writing `articles/index.html`
Reading vignettes/crmPack-jss-paper.Rmd
Writing `articles/crmPack-jss-paper.html`
✖ Missing alt-text in 'vignettes/crmPack-jss-paper.Rmd'
• crmPack-jss-paper_files/figure-html/doseRecommendation-1.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
Reading vignettes/example.Rmd
Writing `articles/example.html`
Missing images in 'vignettes/example.Rmd':
'example_files/figure-html/plot-samplesdata1%20-1.png'
ℹ pkgdown can only use images in 'man/figures' and 'vignettes'
✖ Missing alt-text in 'vignettes/example.Rmd'
• example_files/figure-html/plotdata-1.png
• example_files/figure-html/min-inf-res-1.png
• example_files/figure-html/ggmcmc-1.png
• example_files/figure-html/ggmcmc2-1.png
• example_files/figure-html/plot-model-fit-1.png
• example_files/figure-html/empty-data-1.png
• example_files/figure-html/plot-samplesdata1%20-1.png
• example_files/figure-html/emptydatanoDLTsamples-1.png
• example_files/figure-html/next-best-results-1.png
• example_files/figure-html/next-bestTD-results-1.png
• example_files/figure-html/next-bestTDsamples-results-1.png
• example_files/figure-html/true-def-1.png
• example_files/figure-html/trueDLT-1.png
• example_files/figure-html/third-trial-1.png
• example_files/figure-html/sim-plot-1.png
• example_files/figure-html/sim-sum-plot-1.png
• example_files/figure-html/sim-sum-plot2-1.png
• example_files/figure-html/plotDLTSim-1.png
• example_files/figure-html/plotDLTsampSim-1.png
• example_files/figure-html/DLTSim-plotsummary-1.png
• example_files/figure-html/DLTsampSim-plotsummary-1.png
• example_files/figure-html/sim-future-plot-1.png
• example_files/figure-html/three-sims-plot-1.png
• example_files/figure-html/dual-data-plot-1.png
• example_files/figure-html/dual-conv-1.png
• example_files/figure-html/dual-modelfit-1.png
• example_files/figure-html/dual-variance-1.png
• example_files/figure-html/dual-nextdose-plot-1.png
• example_files/figure-html/dual-sc-plot-1.png
• example_files/figure-html/dual-sims-plot-1.png
• example_files/figure-html/dual-sim-sum-plot-1.png
• example_files/figure-html/plot-samplesdata2loglog-1.png
• example_files/figure-html/plot-samplesdata2Flexi-1.png
• example_files/figure-html/plot-nosamplesEffmodel-1.png
• example_files/figure-html/plotDualResponseNoSamples-1.png
• example_files/figure-html/plotDualResponseSamples-1.png
• example_files/figure-html/plotGain-1.png
• example_files/figure-html/nextbestplot-maxgain-1.png
• example_files/figure-html/nextbest-NextBestMaxGainSamplesplot-1.png
• example_files/figure-html/Truecurves-1.png
• example_files/figure-html/TruecurvesFlexi-1.png
• example_files/figure-html/plotsimulationresults1-1.png
• example_files/figure-html/plotsimulationresults2-1.png
• example_files/figure-html/plotsimulationresults3-1.png
• example_files/figure-html/summarysimulationresults1-1.png
• example_files/figure-html/summarysimulationresults2-1.png
• example_files/figure-html/summarysimulationresults3-1.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
Reading vignettes/migration_from_the_old_crmPack.Rmd
Writing `articles/migration_from_the_old_crmPack.html`
Reading vignettes/ordinal-crm.Rmd
Writing `articles/ordinal-crm.html`
✖ Missing alt-text in 'vignettes/ordinal-crm.Rmd'
• ordinal-crm_files/figure-html/data-ordinal-2-1.png
• ordinal-crm_files/figure-html/plot-1.png
• ordinal-crm_files/figure-html/plot-2.png
• ordinal-crm_files/figure-html/plot-3.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
Reading vignettes/parallel_computing_with_extensions.Rmd
Writing `articles/parallel_computing_with_extensions.html`
Reading vignettes/rolling-crm.Rmd
Writing `articles/rolling-crm.html`
✖ Missing alt-text in 'vignettes/rolling-crm.Rmd'
• rolling-crm_files/figure-html/Diagnose-1.png
• rolling-crm_files/figure-html/Diagnose-2.png
• rolling-crm_files/figure-html/Fit-1.png
• rolling-crm_files/figure-html/Prior-1.png
• rolling-crm_files/figure-html/Recommend-1.png
• rolling-crm_files/figure-html/Truth-1.png
• rolling-crm_files/figure-html/Interpret-1.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
Reading vignettes/simulations.Rmd
Writing `articles/simulations.html`
Reading vignettes/tidy_method.Rmd
Writing `articles/tidy_method.html`
✖ Missing alt-text in 'vignettes/tidy_method.Rmd'
• tidy_method_files/figure-html/unnamed-chunk-7-1.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
Reading vignettes/trial_analysis.Rmd
Writing `articles/trial_analysis.html`
✖ Missing alt-text in 'vignettes/trial_analysis.Rmd'
• trial_analysis_files/figure-html/unnamed-chunk-5-1.png
• trial_analysis_files/figure-html/unnamed-chunk-6-1.png
• trial_analysis_files/figure-html/unnamed-chunk-8-1.png
• trial_analysis_files/figure-html/unnamed-chunk-9-1.png
• trial_analysis_files/figure-html/unnamed-chunk-33-1.png
• trial_analysis_files/figure-html/unnamed-chunk-33-2.png
• trial_analysis_files/figure-html/unnamed-chunk-34-1.png
• trial_analysis_files/figure-html/unnamed-chunk-35-1.png
• trial_analysis_files/figure-html/unnamed-chunk-36-1.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
Reading vignettes/trial_definition.Rmd
Writing `articles/trial_definition.html`
✖ Missing alt-text in 'vignettes/trial_definition.Rmd'
• trial_definition_files/figure-html/unnamed-chunk-5-1.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.

<output deleted>
```

**Comment by Puzzled-Face** [comment]
Fixed:

- [ ] trial_definition
- [ ] trial_analysis
- [ ] rolling_crm
- [ ] ordinal_crm
- [ ] example
- [ ] crmpack-jss-paper
- [ ] favicons
- [ ] DESCRIPTION

**Comment by Puzzled-Face** [comment]
@danielinteractive Please check.  Several of the figures in the rolling CRM vignette were poorly formatted in the original, so I changed their dimensions to make them display better.  Please confirm you are happy with my edits.
